README.md

```diff
@@ -49,6 +53,8 @@ add her to the list of guests in the castle.
 
 Write your code here:
 ```ruby
+town[:residents].delete("Belle")
+town[:castle][:guests].push("Belle")
 # code here
```

Comment:
While this works, were not pushing the same `"Belle"` we deleted. We're just pushing a new string that happens to have the same value, but they are different objects. How might we utilize the delete function output to be used as an input to our push.